src/clj/battlebots/arena/generation.clj

```diff
@@ -11,7 +11,7 @@
 (defn- pos-open
   "returns true of false depending if a given coodinate in a given arena is open"
   [[x y] arena]
-  (= (:open arena-key) (get-in arena [x y])))
+  (= (:open arena-key) (get-in arena [y x])))
```

Comment:
FWIW, it's unfortunate that the arena is organized as a sequence of rows -that prevents the use of the intuitive `(get-in arena coords)` syntax.  Now we have to reverse the contents of the `coords` to use `get-in`.  The only function that benefits from the sequence of rows implementation that I see is the `pprint-arena` function.